components/empty-view/src/types.ts

```diff
@@ -0,0 +1,17 @@
+import { PropsWithChildren, ReactNode } from "react";
+
+import { Illustration } from "./constants";
+
+export type IllustrationKind = keyof typeof Illustration;
+
+export interface ContentProps {
+  readonly body: ReactNode;
+  readonly illustration: IllustrationKind;
+  readonly title: string;
+}
+
+export type EmptyViewProps = PropsWithChildren<{
+  readonly after?: ReactNode;
+  readonly illustration: IllustrationKind;
```

Comment:
Do we want to rename this prop to `kind` or `image` or something else?

Reply:
I think illustration is a good name, since it will result in a selected illustration :)